pkg/topology/query.go

```diff
@@ -331,16 +335,29 @@ func filterComponentsWithDepth(components []*pkg.Component, depth int) []*pkg.Co
 }
 
 func getChecksForComponents() string {
-	return `
-			(SELECT json_agg(checks) from checks LEFT JOIN check_component_relationships ON checks.id = check_component_relationships.check_id WHERE check_component_relationships.component_id = components.id AND check_component_relationships.deleted_at is null   GROUP BY check_component_relationships.component_id) :: jsonb
-			 `
+	return `(
+        SELECT json_agg(checks) FROM checks
+        LEFT JOIN check_component_relationships ON checks.id = check_component_relationships.check_id
+        WHERE check_component_relationships.component_id = components.id AND check_component_relationships.deleted_at IS NULL
+        GROUP BY check_component_relationships.component_id
+    ) :: jsonb`
 }
 
 func getConfigForComponents() string {
-	return `
-       (SELECT json_agg(config_items) from config_items
+	return `(
+        SELECT json_agg(config_items) from config_items
```

Comment:
We shouldn't be using the full config_items table it will return too much data

Reply:
Lets return: `id, name, external_type, config_type, config, cost_per_minute, cost_total_1d, cost_total_7d, cost_total_30d`